### PR TITLE
Addressing items in feedback list from Terraform #1

### PR DESCRIPTION
**Pull request by ja6a**
This commit addresses the following:

- [x] Required attributes should be grouped at the top of the resource or documentation followed by the optional attributes.

- [x] When using TypeInt, setting default to `nil`, is going to result in the value being `0` when it is read. Better to keep this consistent in the schema so there isn't confusion later.

- [x] `project_id` in `skytel_environment` isn't being read or updated, but is also not `ForceNew`. This could lead to being out of sync with with actual environment and confusing diffs. 

- [x] Group standard libraries in imports

- [x] Not a big deal, but naming `vm:update` function something more descriptive would be good to avoid confusion or naming conflicts in the future.

- [x] Leave off <br/> line breaks in documentation

- [x] In the documentation, you can leave arguments off the attributes section.

- [x] CHANGELOG.md should include information on what will be included in the initial release.

See: https://github.com/skytap/terraform-provider-skytap/issues/65

### CODE REVIEW

File: skytap/resource_skytap_vm.go [hunk 1]
```diff
@@ -385,7 +385,7 @@ func resourceSkytapVMDelete(d *schema.ResourceData, meta interface{}) error {
 	return err
 }
 
-func update(d *schema.ResourceData, meta interface{}, forceRunning bool) error {
+func resourceSkytapVMUpdateHelper(d *schema.ResourceData, meta interface{}, forceRunning bool) error {
```
Comment (pegerto) [inline skytap/resource_skytap_vm.go]: Don't think we need to update helper as part of the function name

Comment (ja6a) [reply skytap/resource_skytap_vm.go]: `project_id` has been completely removed. I killed two birds with one stone. There was two items of feedback and the second fixed the first.

Comment (ja6a) [reply skytap/resource_skytap_vm.go]: I have altered the function name.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## 0.9.0 (November 6, 2018)
+
+FEATURES:
+
+* **New Resource:** `skytap_environment` ([#1](https://github.com/skytap/terraform-provider-skytap/issues/1))
```
[the remark this answers is not in the document]
Comment (ja6a) [reply CHANGELOG.md]: done

**Pull request merged**